src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/checked-table-form/checked-table-form.component.ts

```diff
@@ -126,7 +126,7 @@ export class CheckedTableFormComponent implements OnInit {
     // Add or remove the given permission(s) depending on the click event or if no
     // click event is given then add/remove them if they are absent/exist.
     if (
-      (event && event.target['checked']) ||
+      (event && event === true) ||
```

Comment:
if we are getting boolean in event then can we do like this
```suggestion
      if (event || !_.isEqual(permissions.sort(), _.intersection(scopes_permissions[scope], permissions).sort())
```

Reply:
changes are done